### PR TITLE
feat: switch fleet

### PR DESCRIPTION
**Pull request by richard-ramos**
Requires
- https://github.com/status-im/nim-status-go/pull/9
- https://github.com/status-im/status-go/pull/2593

### CODE REVIEW

File: src/app/modules/main/profile_section/advanced/controller.nim [hunk 1]
```diff
@@ -69,6 +64,11 @@ method changeFleetTo*(self: Controller, fleet: string) =
     error "an error occurred, we couldn't set waku version for the fleet"
     return
 
+  if (not self.nodeConfigurationService.setFleet(fleet)):
```
Comment (saledjenic) [inline src/app/modules/main/profile_section/advanced/controller.nim:67]: @richard-ramos does it matter whether we set first fleet or waku version?

File: src/app_service/service/node_configuration/service.nim [hunk 2]
```diff
@@ -151,10 +151,6 @@ method setBloomLevel*(self: Service, bloomLevel: string): bool =
   return false
 
 method setFleet*(self: Service, fleet: string): bool =
-  if(not self.settingsService.saveFleet(fleet)):
```
Comment (saledjenic) [inline src/app_service/service/node_configuration/service.nim:154]: @richard-ramos this line saves fleet to the local db via `settingService` and keeps struct in setting service updated. From the settings service we have methods `getFleet()` and `getFleetAsString()` which are used in multiple places in the app. So are the values read on those places, after calling `setFleet` from this service, going to be accurate?

**Comment by richard-ramos** [comment]
> does it matter whether we set first fleet or waku version?

> are the values read on those places, after calling setFleet from this service, going to be accurate?

@saledjenic , In status-go, my switchFleet function implementation triggers a logout when it's done executing. This is the reason why I set the waku version first, since the RPC server will not be available once switchFleet is executed. In addition to that, the fleet value will be inacurate, but should be okay since we'll logout immedialty anyways.

Would you prefer instead to not logout from switchFleet? I added the logout because switching the fleet in the settings and config leaves the node in an inconsistent state since it will still be connected to the original fleet, so we need to logout immedialty anyways

**Review by saledjenic — APPROVED** [review]
@richard-ramos as long as the official request is not to do it without logout it's safer for us to log out user. :)

**Review by iurimatias — APPROVED** [review]

**Pull request merged**